spec/libsass-todo-issues/issue_947/expected.compact.css

```diff
@@ -0,0 +1,2 @@
+@keyframes test { 10% { color: red; } }
+
```

Comment:
I think this is fine. I force my editor to add a blank line at the end of every file. A lot of specs have this. Also the specs pass on Ruby Sass so I assume all is well.

```
➜  sass-spec git:(master) ruby sass-spec.rb -c sass -s spec/libsass-todo-issues/issue_963
Recursively searching under directory 'spec/libsass-todo-issues/issue_963' for test files to test 'sass' with.
Sass 3.4.9 (Selective Steve)
Run options: --seed 65106

# Running:

....................................................................................................................................................................

Finished in 57.447308s, 2.8548 runs/s, 8.5644 assertions/s.

164 runs, 492 assertions, 0 failures, 0 errors, 0 skips
```